www/scripts/productlist/ListOfProducts.js

```diff
@@ -213,15 +214,46 @@ function (declare, domClass, domConstruct, ItemFileWriteStore, topic,
         + name[0].toUpperCase()
         + '</span></div>';
 
-      if (!item.connected || !item.accessible) {
+
+      var dbStatus = item.databaseStatus
+
+      var db_status_text = ""
+
+      switch(dbStatus) {
```

Comment:
Maybe it would be better to create a converter function (`dbStatusFromCodeToString`) in the `util.js` for this..